tsconfig.shared.json

```
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "incremental": true,
+    "target": "ESnext",
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "lib": [
+        "dom",
+        "ESnext"
+    ],
+    "resolveJsonModule": true,
+    "allowImportingTsExtensions": true,
+    "allowJs": false,
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "declarationMap": true,
+    "jsx": "preserve",
+    "noImplicitAny": true,
+    "noImplicitThis": true,
+    "strictNullChecks": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": false,
+    "strictFunctionTypes": true,
+    "forceConsistentCasingInFileNames": true
```

```suggestion
    "forceConsistentCasingInFileNames": true,
    "verbatimModuleSyntax": true
```

This option _might_ be nice to turn on. It forces you to use `type` imports, allowing bundlers to drop files from which only types are imported.